source/com.microsoft.tfs.core/rest_core/com/microsoft/vss/client/core/model/WrappedException.java

```diff
@@ -9,7 +9,7 @@
 import java.util.List;
 import java.util.Map;
 
-import com.microsoft.vss.client.core.utils.StringUtil;
```

Comment:
Are we sure that this change is correct? Is tfs.util.StringUtil.* a perfect match for vss.client.core.utils.StringUtil.*?